db/migrate/20161214130933_remove_vm_monitoring_id.rb

```diff
@@ -0,0 +1,5 @@
+class RemoveVmMonitoringId < ActiveRecord::Migration[5.0]
+  def change
+    remove_column :atmosphere_virtual_machines, :monitoring_id
```

Comment:
According to http://edgeguides.rubyonrails.org/active_record_migrations.html#using-the-change-method you must supply a datatype when calling `remove_column` from within `change` (otherwise it's irreversible).